sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Generated/DeviceUpdateClient.cs

```diff
@@ -1274,9 +1310,11 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// <summary> Import new update version. </summary>
         /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
         /// <param name="action"> Import update action. Allowed values: &quot;import&quot;. </param>
-        /// <param name="content"> The content to send as the body of the request. </param>
-        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <param name="content"> The content to send as the body of the request. Details of the request body schema are in the Remarks section below. </param>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="action"/> or <paramref name="content"/> is null. </exception>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns cref="Task{Operation}"> The response returned from the service. Details of the response body schema are in the Remarks section below. </returns>
```

Comment:
This method returns: `Task<Operation<BinaryData>>`

Reply:
We cannot generate this in comments with two-level generic type. It is a known issue, I saw many discussion in github:
https://github.com/dotnet/csharplang/issues/401
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2304

Reply:
According to [this](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d313-returns), how about we remove the `cref` part?

Reply:
An explanation of [CDATA] is 

If we generate like this
/// <returns> The Pageable<BinaryData>
The build fails.

If we generate like this
/// <returns> The Pageable{BinaryData}
The rendered doc is like
![image](https://user-images.githubusercontent.com/97225342/174990871-2462c1e8-dc45-4738-b882-7aa46894cfdf.png)

Reply:
Why we can't put `<see cref="Operation{T}"/>`  grammer in text?
We have two-level type [Operation<AsyncPageable<BinaryData>>](https://github.com/archerzz/autorest.csharp/blob/00d2093b0d2d594fc23fdab7d63f1fb8f31cf81a/test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs#L1712).

Reply:
See preview here:
https://raw.githack.com/Azure/azure-sdk-for-net/1a0c08bc10/doc/ApiDocGeneration/Azure.AI.Language.Conversations/docfx-output/docfx_project/site/api/index.html
https://raw.githack.com/Azure/azure-sdk-for-net/1a0c08bc10/doc/ApiDocGeneration/Azure.IoT.DeviceUpdate/docfx-output/docfx_project/site/api/index.html
https://github.com/Azure/azure-sdk-for-net/blob/1a0c08bc10/doc/ApiDocGeneration/Azure.Messaging.WebPubSub/docfx-output/docfx_project/site/api/index.html
https://github.com/Azure/azure-sdk-for-net/blob/1a0c08bc10/doc/ApiDocGeneration/Azure.Security.ConfidentialLedger/docfx-output/docfx_project/site/api/index.html